### ISSUE TITLE
Add Google Analytics to readthedocs

### ISSUE THREAD

**Issue by bendnorman** [catalyst-cooperative/pudl#1463]

**Comment by bendnorman** [comment]
https://docs.readthedocs.io/en/stable/analytics.html

@zaneselvans any idea if this is enabled on our docs right now? 

I'm assuming we have the community version which only stores information for 90 days. Unclear how we can save this data.

**Comment by zaneselvans** [comment]
Yeah, we've got the free community version. The paid versions are kind a pricey, though RTD is a valuable community resource that wouldn't be bad to support.  We do have analytics, and it looks like: [readthedocs_traffic_analytics_catalystcoop-pudl_2021-11-19_2022-02-17.csv](https://github.com/catalyst-cooperative/pudl/files/8085576/readthedocs_traffic_analytics_catalystcoop-pudl_2021-11-19_2022-02-17.csv)

There's a "download all data" on the Admin panel which we could script a scraper to pull, but it doesn't look like the analytics is exposed through an API yet, though [this issue was recently added](https://github.com/readthedocs/readthedocs.org/issues/8891).

Separately there's also data on what search terms people are using within the documentation which could be informative!

We don't have Google Analytics set up, but it looks easy to add if we get an ID. It's been a long time since I played with Google Analytics, but in the past it was hella overwhelming.

**Comment by bendnorman** [comment]
What does adding Google Analytics give us? Is it different from the basic metrics RTD provides?
